dataeng/resources/snowflake-refresh-snowpipe.sh

```diff
@@ -5,4 +5,4 @@ set -ex
 cd $WORKSPACE/analytics-tools/snowflake
 make requirements
 
-python refresh_snowpipe.py -k $KEY_PATH -s $PASSPHRASE -p $PIPE_NAME -t $TABLE_NAME -d $DELAY
+python refresh_snowpipe.py -k $KEY_PATH -s $PASSPHRASE_PATH -u $USER -a $ACCOUNT -p $PIPE_NAME -t $TABLE_NAME -d $DELAY -l $LIMIT
```

Comment:
With so many options being passed, I prefer to see the long argument forms in production scripts.

Reply:
agreed. done.